media/gpu/v4l2/v4l2_device.cc

```diff
@@ -95,6 +95,9 @@ uint32_t V4L2Device::VideoPixelFormatToV4L2PixFmt(VideoPixelFormat format) {
 // static
 uint32_t V4L2Device::VideoCodecProfileToV4L2PixFmt(VideoCodecProfile profile,
                                                    bool slice_based) {
+#if BUILDFLAG(USE_LINUX_V4L2)
+  slice_based = false;
+#endif
   if (profile >= H264PROFILE_MIN && profile <= H264PROFILE_MAX) {
     if (slice_based)
       return V4L2_PIX_FMT_H264_SLICE;
```

Comment:
I'm sorry, I didn't realize this code gets errors as below.
Errors occur if it is built with this layer https://github.com/igel-oss/meta-browser-hwdecode.

| ../../media/gpu/v4l2/v4l2_device.cc:103:14: error: 'V4L2_PIX_FMT_H264_SLICE' was not declared in this scope
|        return V4L2_PIX_FMT_H264_SLICE;
|               ^~~~~~~~~~~~~~~~~~~~~~~
| ../../media/gpu/v4l2/v4l2_device.cc:108:14: error: 'V4L2_PIX_FMT_VP8_FRAME' was not declared in this scope
|        return V4L2_PIX_FMT_VP8_FRAME;
|               ^~~~~~~~~~~~~~~~~~~~~~
| ../../media/gpu/v4l2/v4l2_device.cc:113:14: error: 'V4L2_PIX_FMT_VP9_FRAME' was not declared in this scope
|        return V4L2_PIX_FMT_VP9_FRAME;
|               ^~~~~~~~~~~~~~~~~~~~~~
| ../../media/gpu/v4l2/v4l2_device.cc:115:14: error: 'V4L2_PIX_FMT_VP9' was not declared in this scope
|        return V4L2_PIX_FMT_VP9;
|               ^~~~~~~~~~~~~~~~

It should be changed to avoid compiling these lines.
I guess Damian avoided this build error as below.
https://github.com/igel-oss/meta-browser-hwdecode/blob/master/recipes-chromium/chromium/files/0001-Add-support-for-V4L2VDA-on-Linux.patch#L227
Which is better? creating a new pull request or fixing on this pull request?

Reply:
please create a new commit, which starts with "fixup!" word and then the commit title.

Reply:
I see. I'll create it.